src/components/Chat/Chat.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect, useContext } from "react";
+import ChatArea from "./ChatArea";
+import ChatForm from "./ChatForm";
+import SearchBar from "../SearchBar/SearchBar";
+import "./Chat.css";
+import * as UserAPI from "../../UserAPI";
+import APIHeaders from "../../APIContext";
+import axios from "axios";
+import AllUsers from "../../AllUsersContext";
+
+function Chat() {
+  const [header, setHeader] = useContext(APIHeaders);
+  const [allUsers, setAllUsers] = useContext(AllUsers);
+  const [chatWith, setChatWith] = useState("");
+  const [userSearch, setUserSearch] = useState("");
+
+  useEffect(() => {
+    console.log(chatWith);
+  }, [chatWith]);
+  // LOG IN BUTTON LANG DITO FOR TESTING AND GET USERS - START
+  const logIn = () => {
+    const url = "http://206.189.91.54//api/v1/auth/";
+
+    axios
+      .post(url + "sign_in", {
+        email: "testhello@test.com",
+        password: "Hello12345",
+      })
+      .then((res) => {
+        console.log("success: " + res);
+        setHeader({
+          "access-token": res.headers["access-token"],
+          client: res.headers["client"],
+          expiry: res.headers["expiry"],
+          uid: res.headers["uid"],
+        });
+
+        console.log("headers: " + header);
+        console.log("res.headers: " + res.headers["access-token"]);
+      })
+      .catch((e) => {
+        console.log("error: " + e);
+      });
+  };
+
+  const recentlyDms = () => {
+    UserAPI.getRecent(header)
+      .then((res) => {
+        console.log(res);
+      })
+      .catch((e) => {
+        console.log("What is recently DMS");
+      });
+  };
+  // LOG IN BUTTON LANG DITO FOR TESTING AND GET USERS - END

```

Comment:
Great job in adding comments. This will be useful when we start to figure out how to pass this upon sending messages.